ui-modules/blueprint-composer/app/views/main/graphical/edit/add/add.js

```diff
@@ -27,17 +27,18 @@ export const graphicalEditAddState = {
     name: 'main.graphical.edit.add',
     url: '/add/:family?configKey',
     template: template,
-    controller: ['$scope', '$filter', '$state', '$stateParams', 'blueprintService', GraphicalEditAddController],
+    controller: ['$scope', '$filter', '$state', '$stateParams', 'blueprintService', 'paletteService', GraphicalEditAddController],
     controllerAs: 'vm',
 };
 
-export function GraphicalEditAddController($scope, $filter, $state, $stateParams, blueprintService) {
+export function GraphicalEditAddController($scope, $filter, $state, $stateParams, blueprintService, paletteService) {
     switch ($stateParams.family) {
         case EntityFamily.ENTITY.id.toLowerCase():
             $scope.family = EntityFamily.ENTITY;
             break;
         case EntityFamily.SPEC.id.toLowerCase():
             $scope.family = EntityFamily.SPEC;
+            $scope.familiesToShow = [ EntityFamily.ENTITY, EntityFamily.SPEC ];
```

Comment:
Seems to be used only in the JS code so would be better to be a local var (i.e. avoiding adding stuff to the `$scope` as a general rule of thumb)

Reply:
no, it's used in the html

Reply:
I cannot find any reference if this on templates

Reply:
https://github.com/apache/brooklyn-ui/pull/93/files#diff-ed969f9a529823af9b9282a89cc5ebc1R31